components/issue-list-item.tsx

```diff
@@ -73,94 +98,44 @@ export default function IssueListItem({
-                isActive && "-40"
-              }`}
-            >
-              {+bountyAmount >= 1.e-6 && formatNumberToNScale(bountyAmount?.toFixed()) || 
-                bountyAmount?.toExponential()}{" "}
-                
-              <label
-                className={`caption-small text-uppercase ${
-                  !isActive ? "text-primary" : "text-white-40"
-                }`}
-              >
-                {issue?.token?.symbol || t("common:misc.token")}
-              </label>
-            </span>
-          </div>
-        )}
-        {isFundingRequest && issue?.fundedAmount?.isLessThan(issue?.fundingAmount) && (
-          <>
-            <div className={`p-0 col-md-6 col-10 mt-1 ${isMobile && "pt-1"}`}>
-              <div className="bg-dark-gray w-100 issue-funding-progress">
-                <div
-                  className={`bg-primary issue-funding-progress`}
-                  style={{ width: `${percentage}%` }}
+      <CardItem onClick={handleClickCard}>
+        <>
+          <div className="d-flex justify-content-between">
+            <div className="bg-dark-gray p-1 border-radius-8">
+              {issue?.network?.logoIcon && (
+                <img
+                  src={`${state.Settings?.urls?.ipfs}/${issue?.network?.logoIcon}`}
+                  width={14}
+                  height={14}
+                  className="ms-1 me-2"
                 />
-              </div>
+              )}
+              <span className="caption-small me-1 text-uppercase text-truncate">
+                {issue?.network?.name}
+              </span>
             </div>
-            <div
-              className={`issue-percentage-text caption-small py-0 pe-0 ps-1 pb-1 col-2 col-md-2 text-white
-              ${isMobile && "pt-1"}`}
-            >
-              {percentage}%
-            </div>
-          </>
-        )}
-      </div>
-      </OverlayTrigger>
+
+            <BountyStatusInfo issueState={issueState} className="mt-1 px-2 " />
+          </div>
+          <div className="text-truncate mb-2 mt-4">{issue?.title}</div>
+          <div className="issue-body text-white-40 text-break text-truncate mb-3" >
+            {issue?.body}
+          </div>
+          <IssueAmountInfo issue={issue} size={size} />
+        </>
+      </CardItem>
     );
   }
 
+
   return (
-    <div
-      className="bg-shadow list-item p-3"
-      onClick={() => {
-        if (xClick) return xClick();
-
-        router.push({
-          pathname: "/[network]/bounty",
-          query: {
-            id: issue?.githubId,
-            repoId: issue?.repository_id,
-            network: state.Service?.network?.lastVisited,
-          },
-        });
-      }}
-    >
+    <CardItem onClick={handleClickCard}>
       <div className="row align-center">
         <div className="col-md-10 mb-3 mb-md-0">
           <h4 className="h4 text-truncate mb-3">
-            <span className="text-white-40 me-2">#{issue?.githubId}</span>
+            <Tag/>
```

Comment:
This name will certainly raise conflicts with the tags coming in from the other story, change it to better naming would be `creatorTag()` that way we know _what_ will be inside the tag